news/2019-09-30-beatmap-spotlights-summer-2019.md

```diff
@@ -0,0 +1,386 @@
+---
+layout: post
+title: "Beatmap Spotlights: Summer 2019"
+date: 2019-09-30 10:00:00 +0000
+---
+
+Even if the hot season flew away and colder winds are blowing, we still have something in our pocket to keep you warm. Our selection from the hotest season is here for you the spend some time with. Come take a look and earn a fancy new medal for yourself by playing them all!
+
+![](/wiki/shared/news/2019-09-30-beatmap-spotlights-summer-2019/header.jpg)
+
+The beatmap charts for the Summer Spotlights can be found [here](https://osu.ppy.sh/rankings/taiko/charts?spotlight=267). Their respective packs are available at the [Beatmap Pack Listing](https://osu.ppy.sh/beatmaps/packs?type=chart) for easier access. **The Summer 2019 Spotlights rankings will end on the 15th of December, 2019.**
+
+## Spring Spotlights results
+
+First of all, congratulations to the winners of the Seasonal Spotlights: Summer 2019. They will receive 3, 2, and 1 month of supporter for 1st, 2nd, and 3rd place respectively for their efforts!
+
+- **osu!:** [Toy](https://osu.ppy.sh/users/2757689), [Woey](https://osu.ppy.sh/users/3792472) and [xxluizxx47](https://osu.ppy.sh/users/4687701)
+- **osu!taiko:** [Jaye](https://osu.ppy.sh/users/4841352), [AlvaroYL](https://osu.ppy.sh/users/6333166) and [Whulf](https://osu.ppy.sh/users/11601107)
+- **osu!catch:** [RAMPAGE88](https://osu.ppy.sh/users/448547), [Predominador](https://osu.ppy.sh/users/4568537) and [Chatie](https://osu.ppy.sh/users/6524765)
+- **osu!mania:** [\[Crz\]Satori](https://osu.ppy.sh/users/7082178), [Stink God](https://osu.ppy.sh/users/7381289) and [Cryolien](https://osu.ppy.sh/users/1626983)
+
+
+# Navigation
+
+- <a href="#osu">osu!</a>
+- <a href="#taiko">osu!taiko</a>
+- <a href="#catch">osu!catch</a>
+- <a href="#mania">osu!mania</a>
+
+# Summer 2019 Spotlights
+
+## <a name="osu" id="osu"></a>osu!
+
+[![](/wiki/shared/news/2019-09-30-beatmap-spotlights-summer-2019/osu/hide.jpg)](https://osu.ppy.sh/beatmapsets/972932)
+
+What better way to introduce a new Featured Artist's music than with [Mirash](https://osu.ppy.sh/users/2841009)'s set of "Hide And Seek". This wonderfully jazzy piano track is all "get-up-and-go" featuring flurries of unconventional cut-stream patterns in the top difficulty. 
+
+Lots of rhythmic variation allow both Mirash and [RLC](https://osu.ppy.sh/users/1047883) to play around with their patterning throughout the set's difficulty spread, leaving the player with a highly diverse set of levels to play through. All of this is complimented beautifully by [Noffy](https://osu.ppy.sh/users/1541323)'s exquisite storyboard, earning this map the top spot in this season's spotlights.
+
+_written by [Toy](https://osu.ppy.sh/users/2757689)_
```

Comment:
move these to the beginning of each description, instead of the end
also use `*` instead of `_`

Reply:
I think that it is fine at the bottom, that way it stays consistent with our previous article